index.html

```diff
@@ -0,0 +1,1942 @@
+				  </div>
+				  <div id="special-item-3" class="special-item-right">
+					<div class="content">
+					  <div class="head-story">
+						<article>
+						  <div class="article-wrap">
+							<div class="category">
+							  <a href="#">sponsored insight</a>
+							</div>
+							<div class="image">
+							  <a href="#"><span id="special-3"></span>
+							  </a>
+							</div>
+						  </div>
+						  <div class="article-summary">
+							<h3 class="header-3">
+							  <a href="#">Meet Jerry O'Connell in NY</a>
+							</h3>
+							Win a trip for two to the Big Apple for a special meet and greet!
+						  </div>
+						</article>
+					  </div>
+	
+					</div>
+				  </div>
+				  <div id="special-item-4" class="special-item-right">
+					<div class="content">
+					  <div class="head-story">
+						<article>
+						  <div class="article-wrap">
+							<div class="category">
+							  <a href="#">sponsored insight</a>
+							</div>
+							<div class="image">
+							  <a href="#"><span id="special-4"></span>
+							  </a>
+							</div>
+						  </div>
+						  <div class="article-summary">
+							<h3 class="header-3">
+							  <a href="#">If You or a Loved One Has Been Injured, Who Can Champion Your Cause?</a>
+							</h3>
+							Logic dictates that when you need a good personal injury attorney, you’re going to be in a
+							stressful,
+							vulnerable and overwhelming situation. It is vital that you speak to an attorney who can ensure
+							that
+							you
+							are not taken advantage of. We look at some of the leaders in the field.
+						  </div>
+						</article>
+					  </div>
+	
+					</div>
+				  </div>
+				</div>
+			  </div>
+			  <div class="col2">
+				<div class="col-wrap d-flex flex-column tp-story">
+				  <div class="top-story-content order-1">
+					<div class="content">
+					  <div class="head-story">
+						<div class="title">
+						  <p>Top Story</p>
+						</div>
+						<article>
+						  <div class="article-wrap">
+							<div class="category">
+							  <a href="#">Analysis</a>
+							</div>
+							<div class="image">
+							  <a href="#">
+								<span>
+								  <img class="img-fluid" src="images/top-story1.jpg" alt="Trump Iran descalation">
+								</span>
+							  </a>
+							</div>
+						  </div>
+						  <h1 class="header-2">
+							<a href="#">With a New Weapon in Trump's Hands, the Iran Crisis Risks Going Nuclear</a>
+						  </h1>
+						  <div class="article-summary">
+							Exclusive: Now that the U.S. has deployed a new nuclear weapon, the options presented to
+							President
+							Donald
+							Trump to respond to Tehran will automatically include a nuclear option.
+						  </div>
+	
+						</article>
+	
+					  </div>
+					</div>
+				  </div>
+				  <div class="more-story-content order-3">
+					<div class="content">
+					  <div class="head-story">
+						<p>more stories</p>
+					  </div>
+					  <article class="more-story-article">
+						<div class="image">
```

Comment:
- [ ] The images of this section are larger than the original page. Check all breakpoints and make the necessary adjustments.

Reply:
Already Did